src/WidgetMetadataEditor.jsx

```diff
@@ -0,0 +1,47 @@
+const widgetPath = props.widgetPath;
+const onChange = props.onChange;
+
+let metadata = Social.getr(`${widgetPath}/metadata`);
+
+if (metadata === null) {
+  return "Loading";
+}
+
+return (
+  <Widget
+    key={widgetPath}
+    src="mob.near/widget/MetadataEditor"
+    props={{
+      initialMetadata: metadata,
+      onChange,
+      options: {
+        name: { label: "Title" },
+        image: { label: "Icon" },
+        description: { label: "Description" },
+        tags: {
+          label: (
+            <>
+              Tags{" "}
+              <span className="text-secondary">
+                {" "}
+                (add relevant tags to ensure the best discoverability of your
+                app/component)
+              </span>
+            </>
```

Comment:
Only this part were changed according to this [task](https://pagodaplatform.atlassian.net/browse/DEC-1111)